### PR TITLE
[Performance, Triton] Optimize over mask compute to tl.load in fused_moe_kernel

### PR DESCRIPTION
**Pull request by HaiShaw**
## Motivation

Test shows `~0.5%` boost to prefill, `~1.0%` boost to median decode throughput over Grok-1 with `b32/i1023/o256` settings.

## Modifications

`fused_moe_kernel`: simplify the mask part for even K BLOCK sizes
`decode_attention`: adjust kernel args

## Checklist

- [+] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [+] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [+] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: python/sglang/srt/layers/fused_moe/fused_moe.py [hunk 1]
```diff
@@ -130,24 +131,42 @@ def fused_moe_kernel(
...
+    if even_Ks:
+        for k in range(0, tl.cdiv(K, BLOCK_SIZE_K)):
+            # Load the next block of A and B, generate a mask by checking the
+            # K dimension.
+            a = tl.load(
+                a_ptrs,
+                mask=token_mask[:, None],
+                other=0.0,
+            )
+            b = tl.load(b_ptrs)
```
Comment (merrymercy) [inline python/sglang/srt/layers/fused_moe/fused_moe.py:143]: it seems only these two lines are different, please put `if even_Ks` here.

**Review by merrymercy — CHANGES_REQUESTED** [review]

File: python/sglang/srt/layers/fused_moe/fused_moe.py (hunk 1 above)
Comment (HaiShaw) [reply python/sglang/srt/layers/fused_moe/fused_moe.py:143]: I was considering to do less branch within for loop, what do you think?

Comment (merrymercy) [reply python/sglang/srt/layers/fused_moe/fused_moe.py:143]: It is a const expr so the compiler should be able to simplify it. Can you verify the perf?

Comment (HaiShaw) [reply python/sglang/srt/layers/fused_moe/fused_moe.py:143]: Verifying now.

Comment (HaiShaw) [reply python/sglang/srt/layers/fused_moe/fused_moe.py:143]: @merrymercy , moving `even_Ks` within for-loop does not cause much perf difference on ROCm.
I will make the changes accordingly.

**Pull request merged**